### ISSUE TITLE
Doesn't boot yet, mehh....

### ISSUE THREAD

**Issue by visi0nary** [visi0nary/android_kernel_xiaomi_msm8937_caf_upstream#2]

**Comment by nervehammer** [comment]
Hope it will boot soon :P Good luck @visi0nary, @Klozz and @nichcream

**Comment by visi0nary** [comment]
Actually I think the main problem is the LCD dts file that holds us from booting it properly. We'll find a way to fix it though

**Comment by nervehammer** [comment]
damn this LCD :/ I wonder how los 14.1 with 3.18.31 and enforcing se linux booted :p

**Comment by visi0nary** [comment]
Yeah well I think once the kernel boots the hardest work is done ;) Tailoring an SELinux context is less magic IMHO
